test/react16_suggestors/react_dom_render_migrator_test.dart

```diff
@@ -458,6 +502,34 @@ main() {
       );
     });
 
+    test('no react_dom.dart import but usage has namespace', () {
+      testSuggestor(expectedPatchCount: 2, input: '''
+          part of 'a_file.dart';
+
+          main() {
+            react_dom.render(Foo()(), mountNode);
+          }
+        ''', expectedOutput: '''
+          part of 'a_file.dart';
+
+          main() {
+            react_dom.render(ErrorBoundary()(Foo()()), mountNode);
+          }
+        ''');
+    });
+
+    test('no react_dom.dart import but usage has namespace', () {
```

Comment:
#nit this test has the same description as the one before it

Reply:
nice catch